src/main/java/com/dwbtest/App.java

```diff
@@ -0,0 +1,12 @@
+package com.dwbtest;
+
+/**
+ * Hello world!
+ *
+ */
+public class App {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Utility classes should not have a public or default constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck)